ruby/spec/authentication_spec.rb

```diff
@@ -5,7 +5,7 @@
 require 'base64'
 require_relative '../app/middleware/auth'
 require_relative '../app/routes/health'
-
+require_relative './helpers/token'
```

Comment:
Is `token` good naming? WDYT about `helpers/auth_header`? So renaming the file to `auth_header.rb`?